leakcanary-android-core/src/main/java/leakcanary/internal/LeakDirectoryProvider.kt

```diff
@@ -137,7 +137,7 @@ internal class LeakDirectoryProvider constructor(
   }
 
   private fun appStorageDirectory(): File {
-    val appFilesDirectory = context.filesDir
+    val appFilesDirectory = context.cacheDir
```

Comment:
@pforhan this one's for you and your emulators with no disk space ;)

Reply:
aww, you're too kind